docs/installation/autoware/docker-installation.md

```diff
@@ -53,7 +53,7 @@ For more launch options you can edit the launch command with `--launch-cmd` opti
 
 #### Run Autoware simulator
 
-Inside the container, you can run the Autoware simulation by following these tutorials:
+Inside the container, you can run Autoware simulation by following these tutorials:
```

Comment:
same, "the ... simulation", so "the Autoware simulation" was correct

Reply:
fixed in https://github.com/autowarefoundation/autoware-documentation/pull/516/commits/7b93b72fb23b38494d11fb884550a802eee021c3